coap-gateway/service/observation/deviceObserver.go

```diff
@@ -51,11 +53,13 @@ type DeviceObserver struct {
 }
 
 type DeviceObserverConfig struct {
-	Logger                     log.Logger
-	ObservationType            ObservationType
-	TwinEnabled                bool
-	TwinEnabledSet             bool
-	RequireBatchObserveEnabled bool
+	Logger                       log.Logger
+	ObservationType              ObservationType
+	TwinEnabled                  bool
+	TwinEnabledSet               bool
+	UseETags                     bool
+	RequireBatchObserveEnabled   bool
+	LimitBatchObserveLatestETags uint32
```

Comment:
What is this?

Reply:
This should be equal to configuration of maxETagsCount, because more you don't need.

Reply:
fixed